spec/fake_braintree/address_spec.rb

```diff
@@ -1,14 +1,21 @@
 require 'spec_helper'
 
 describe "Braintree::Address.create" do
-  it "successfully creates address with valid data" do
-    customer_response = create_customer
+  let(:customer) { create_customer.customer }
 
+  it "successfully creates address with valid data" do
     address_response = Braintree::Address.create(
-      customer_id: customer_response.customer.id,
+      customer_id: customer.id,
       postal_code: 30339
     )
 
     expect(address_response).to be_success
   end
+
+  it "sets the creation time" do
+    address = Braintree::Address.create(customer_id: customer.id).address
+
+    creation_time = Time.parse(address.created_at)
+    expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.